.npmrc

```
@@ -1,3 +1,4 @@
 tag-version-prefix = ""
 message = "Release %s"
 preid = "beta"
+loglevel=silent
```

fix: this should be error level because we still want to see npm errors, but we don't want to see info and warnings. This is to make analysis script piping with jq work when running `npm run api-usage-analyze | jq`.
```suggestion
loglevel=error
```